test/specs/reference/tests/12.reference_values_edit.js

```diff
@@ -234,6 +234,105 @@ exports.execute = function (options) {
+                tableName = "jsontest_table",
+                lowerLimit = 1001,
+                upperLimit = 2001,
+                limit = 6;
+
+            var multipleEntityUri=options.url + "/catalog/" + catalog_id + "/entity/" + schemaName + ":"+ tableName ;
+
+            var reference, page, tuples,url;
+            
+            var chaiseURL = "https://dev.isrd.isi.edu/chaise";
+            var recordURL = chaiseURL + "/record";
+            var record2URL = chaiseURL + "/record-two";
+            var viewerURL = chaiseURL + "/viewer";
+            var searchURL = chaiseURL + "/search";
+            var recordsetURL = chaiseURL + "/recordset";
+
+            var appLinkFn = function (tag, location) {
+                
+                switch (tag) {
+                    case "tag:isrd.isi.edu,2016:chaise:record":
+                        url = recordURL;
+                        break;
+                    case "tag:isrd.isi.edu,2016:chaise:record-two":
+                        url = record2URL;
+                        break;
+                    case "tag:isrd.isi.edu,2016:chaise:viewer":
+                        url = viewerURL;
+                        break;
+                    case "tag:isrd.isi.edu,2016:chaise:search":
+                        url = searchURL;
+                        break;
+                    case "tag:isrd.isi.edu,2016:chaise:recordset":
+                        url = recordsetURL;
+                        break;
+                    default:
+                        url = recordURL;
+                        break;
+                }
+
+                url = url + "/" + location.path;
+
+                return url;
+            };
+
+
+            beforeAll(function(done) {
+                
+                // Fetch the entities beforehand
+                options.ermRest.resolve(multipleEntityUri).then(function (response) {
+                    reference = response;
+                    expect(reference).toEqual(jasmine.any(Object));
+                    reference = reference.contextualize.entryEdit;
+                    return reference.read(limit);
+                }).then(function (response) {
+                    page = response;
+                    
+                    expect(page).toEqual(jasmine.any(Object));
+                    expect(page._data.length).toBe(limit);
+                    
+                    expect(page.tuples).toBeDefined();
+                    tuples = page.tuples;
+                    expect(tuples.length).toBe(limit);
+                    
+                    done();
+                }, function (err) {
+                    console.dir(err);
+                    done.fail();
+                }).catch(function(err) {
+                    console.dir(err);
+                    done.fail();
+                });
+                options.ermRest.appLinkFn(appLinkFn);
+            });
+            
+            it("Testing for EDIT JSON and JSONB tuples values", function() {
```

Comment:
Change this to something like "should return the expected values." to be a more meaningful log.